src/api/spec/factories/project.rb

```diff
@@ -77,11 +77,21 @@
 
       transient do
         target_project nil
+        create_patchinfo false
+        maintainer nil
       end
 
       after(:create) do |project, evaluator|
         create(:maintainance_project_attrib, project: project)
-        create(:maintained_project, project: evaluator.target_project, maintenance_project: project) if evaluator.target_project
+        if evaluator.target_project
+          create(:maintained_project, project: evaluator.target_project, maintenance_project: project)
+          CONFIG['global_write_through'] ? project.store : project.save!
+        end
+        if evaluator.create_patchinfo
```

Comment:
I found that same code [here](https://github.com/openSUSE/open-build-service/blob/af51d523be0696172682bb32d31f4da509d18a07/src/api/spec/factories/project.rb#L22-L35). Must I change it in both places?

I case I use your suggestion I would like to use the variable name `create_patchinfo_maintainer`, instead of `maintainer`.